examples/blog/pages/index.mdx

```diff
@@ -0,0 +1,56 @@
+---
+type: page
+title: About
+date: 2021-03-19
+---
+
+# Your Name
+
+Hey, I'm a Senior Software Engineer at Company. I enjoy working with Next.js and crafting beautiful front-end experiences.
+
+This portfolio is built with **Next.js** and a library called [Nextra](https://nextra.vercel.app/). It allows you to write Markdown and focus on the _content_ of your portfolio.
+
+[**Deploy your own**](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/next.js/tree/canary/examples/blog&project-name=portfolio&repository-name=portfolio) in a few minutes.
+
+---
+
+<div>
+  Twitter [@yourname](https://twitter.com/yourname)
+  <br />
+  GitHub [@yourname](https://github.com/yourname)
+  <br />
+  Instagram [@yourname](https://instagram.com/yourname)
+  <br />
+  Email your@name.com
+</div>
+
+<br />
+
+<details><summary children="Resume"/>
```

Comment:
I think this `<details>` toggle is overkill for most users (and it introduces styled-jsx), maybe we can get rid of it to make it simpler?

Reply:
Works for me!